main.py

```diff
@@ -3,48 +3,56 @@
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 import discord
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_slash.utils import manage_commands
 import datetime
 import json
 import re
 import html2text
+import sys
 
 def jsonFromFile(filePath):
   with open(filePath,"r",encoding="utf8") as file:
-    return json.loads(file.read())
+    return json.load(file)
 
 #top secret stuff !
-pws = jsonFromFile("pws.json")
-pw = pws["pw"]
-username = pws["username"]
-token = pws["token"]
+settings = jsonFromFile("settings.json")
+pw = settings["password"]
+username = settings["username"]
+token = settings["token"]
+#admin ids
+admins = settings["admins"]
+# server id (singular)
+guild_id = settings["guild_id"]
 
 #Stuff Stack Overflow told me to put to stop crashes ¯\_(.-.)_/¯
 browser_options = Options()
 browser_options.add_argument('--no-sandbox')
 browser_options.add_argument('--disable-dev-shm-usage')
 
 #headless !!!
-#browser_options.add_argument("headless")
+if "headless" in settings and settings["headless"]:
+  browser_options.add_argument("headless")
 
 #Data
-links = jsonFromFile("links.json")
+data = jsonFromFile("data.json")
+links = data["links"]
 #schedule in 24h clock
-times = jsonFromFile("times.json")
+times = data["times"]
 #channel to ping upon start
-pingChannel = jsonFromFile("channel.json")
-#admin ids  v THIS ME :) v
-admins = [414212931023011855]
-
-def cacheFile(obj,filePath):
-  with open(filePath,"w") as file:
-    file.write(json.dumps(obj))
+pingChannel = data["channel"]
 
 def cache():
-  cacheFile(links,"links.json")
-  cacheFile(times,"times.json")
-  cacheFile(pingChannel,"channel.json")
+  with open("data.json", "w") as file:
+    json.dump({
+      "links": links,
+      "times": times,
+      "channel": pingChannel
+    }, file, indent=4)
 
-client = discord.Client() #get discord
+client = commands.Bot(command_prefix=".", intents=discord.Intents.all()) #get discord
```

Comment:
does a prefix of nothing work? If so please use it instead. 
```suggestion
client = commands.Bot(command_prefix="", intents=discord.Intents.all()) #get discord
```

Reply:
Note that without subcommand support you will be limited to a string without spaces for command invocations.